src/agent/agent_info/src/agent_info_persistance.cpp

```diff
@@ -75,6 +86,22 @@ void AgentInfoPersistance::CreateAgentInfoTable()
     }
 }
 
+void AgentInfoPersistance::CreateAgentGroupTable()
+{
+    try
+    {
+        m_db->exec("CREATE TABLE IF NOT EXISTS " + AGENT_GROUP_TABLE_NAME +
```

Comment:
I think you might be able to use `AGENT_INFO_TABLE_NAME` in the `CreateAgentInfoTable()` function, similar to how `AGENT_GROUP_TABLE_NAME` is used here.

Reply:
Done!